src/machine/asm/execute.S

```diff
@@ -177,26 +179,54 @@
   CALL_MEMSET; \
   POSTCALL;
 
+#ifdef IS_WINDOWS
 #define RANDOM_MEMORY \
   PREPCALL; \
+  mov CHAOS_SEED, ARG1; \
+  call srand; \
```

Comment:
I think we might only need to call `srand` once when we initialize the VM, right now the implementation calls srand before each page initialization, this would result in the fact that each page is filled with exactly the same data, correct?

Reply:
The seed used for the second `srand()` is the last data generated by the first `srand()` + `rand()`

Reply:
I see, in that sense this works.